wagtail_localize_rws_languagecloud/signals.py

```diff
@@ -0,0 +1,4 @@
+from django.dispatch import Signal
+
+
+translation_imported = Signal()
```

Comment:
Nice. I like this approach of registering a signal so the user can hook into it in a generic way :+1: